services/vision/vision.go

```diff
@@ -125,10 +126,21 @@ func FindFirstName(r robot.Robot) string {
 	return ""
 }
 
-// FirstFromRobot returns the first vision service in this robot.
+// FirstFromRobot returns the first vision service on this robot.
 func FirstFromRobot(r robot.Robot) (Service, error) {
-	name := FindFirstName(r)
-	return FromRobot(r, name)
+	return FirstFromLocalRobot(r) 
+}
+
+// FirstFromRobotMain returns the first vision service on this main robot.
+// This will specifically ignore remote resources.
+func FirstFromLocalRobot(r robot.Robot) (Service, error) {
```

Comment:
nit : comment doesn't match the name of the function

Reply:
Also done lol